modules/analytics/src/views/full/index.tsx

```diff
@@ -514,7 +514,7 @@ const Analytics: FC<any> = ({ bp }) => {
                 key={i.language}
                 value={i.value}
                 color="#F2B824"
-                name={`${lang.tr(`isoLangs.${i.language}.name`)}: ${i.value}`}
+                name={`${i.language !== 'n/a' ? lang.tr(`isoLangs.${i.language}.name`) : 'Unknown'}: ${i.value}`}
```

Comment:
Please use i18n instead of hard-coding 'Unknown' unless you consider it's unnecessary?

Reply:
+1. you could simply add `n/a` in the keys of translations in fact.. or if it doesnt support slashes, replace n/a with unknown and add the translation

Reply:
yeah true, will do, forgot about translation